### DISCUSSION TITLE
Hubble parameter is not a constant

### DISCUSSION THREAD

**Discussion by VitalyPetrikov** [a-cosmology-group/acg#123] [Astrophysics and Physics]
Let's pre-designate the main parameters (properties) of the subject under consideration.
The Hubble parameter cannot be a constant for the following reasons:
The Hubble parameter is drawn on speed-distance schemes as an averaging of too much spread from the average value, and therefore, even with time (already 100 years), it does not come to a certain undisputed value. At average radii, the Hubble parameter is somehow determined, and with an increase in the speeds of the observed galaxies, all sorts of tricks from the universe are used to preserve it.
The average distances between galaxies are approximately the same throughout cosmos in any direction and are determined by the procedure for the formation of galaxies (see sections 2 and 6). That is, the increase in distances with each farther galaxy is practically linear. Of course, there are regions in cosmos with a thicker existence of galaxies, as, accordingly, there are regions with very large voids, but these facts only confirm the rule. But the velocities of galaxies, when approaching light speed, change more and more to a lesser extent.
And in this regard, the Hubble parameter is curved. It is clear from Figure 3 that if the counting of distances from the center (on the scale of distances) at radii of about 1/3 of the radius of modern cosmos is linear, then the counting of the speed of galaxies on the scale of speeds is heated in place.
Hubble parameter is curved in principle and its curvature is approximately determined by the simplified formula given in the essay (section 4 and Fig. 3). At the same time, all sorts of tricks inspired by all sorts of cunning, followers of the theories of the Big Bang, become useless.
In general, it makes no sense to attach great importance to the Hubble parameter, since it practically does not affect anything and nothing follows from his knowledge. The Hubble parameter is a simple statement of fact. Much more interesting is the question of the area of location of galaxies having near-light speed. 

https://github.com/orgs/a-cosmology-group/discussions/115

**Comment by RedshiftDrift** [comment]
> The Hubble parameter is a simple statement of fact.

Yes, but its incorrect interpretation as a measurement of the "rate of expanding space" is the pillar of modern cosmology.  Any anomaly with the Hubble parameter is a sign of a weakness in Big Bang cosmology.

**Comment by VitalyPetrikov** [comment]
How is this "misinterpretation ... is a pillar of modern cosmology"? This place would like more details.

**Reply by RedshiftDrift** [reply]
Pillars of cosmology:
1- [the expansion of the universe](https://en.wikipedia.org/wiki/Big_Bang#Observational_evidence) according to Hubble's law (***as indicated by the redshifts of galaxies***),
2- etc.

However, the *redshift of galaxies as caused by expansion* is a misinterpretation.  In reality, the cosmological redshift is caused by a tired-light effect.  There is no expansion of space.

**Reply by HanDeBruijn** [reply]
@RedshiftDrift . Two points.

>the cosmological redshift is caused by a tired-light effect

I wouldn't be so sure about that.

>There is no expansion of space.

With the Variable Mass Theory, the expansion of (empty) space is observationally equivalent with [Lengh Contraction](http://www.alternatievewiskunde.nl/cosmology/lengte.htm).
